scripts/generate-types.js

```diff
@@ -85,9 +85,25 @@ type Repository = components["schemas"]["full-repository"]
 
     await copyFile("LICENSE", `packages/${packageName}/LICENSE`);
 
+    const schemaTS = await openapiTS(`cache/${name}.json`, {
+      transform(schemaObject, metadata) {
+        if (
+          schemaObject.format === "binary" &&
+          metadata.path.endsWith("application/octet-stream")
+        ) {
+          // Make sure that file upload endpoints don't use 'string' type for binary data
+          // Use some common types that can represent binary data in various environments
+          return schemaObject.nullable
+            ? "string | File | UInt8Array | Blob | null"
+            : "string | File | UInt8Array | Blob";
```

Comment:
I think we're missing `Buffer`. Typical usage would be reading the file with `fs.readFileSync`

Reply:
`Buffer` is a subclass of `UInt8Array`, it will match with that

Reply:
LGTM then! 🎉